app/controllers/SchemePreferencesController.scala

```diff
@@ -29,30 +29,38 @@ class SchemePreferencesController(applicationClient: ApplicationClient, schemeCl
 
   def present = CSRSecureAppAction(SchemesRole) { implicit request =>
     implicit user =>
+      val civilServant = isCivilServant(user.application.fastPassDetails)
       schemeClient.getSchemePreferences(user.application.applicationId).map { selectedSchemes =>
-        Ok(views.html.application.schemePreferences.schemeSelection(form.fill(selectedSchemes)))
+        Ok(views.html.application.schemePreferences.schemeSelection(civilServant, form.fill(selectedSchemes)))
       }.recover {
         case e: SchemePreferencesNotFound =>
-          Ok(views.html.application.schemePreferences.schemeSelection(form))
+          Ok(views.html.application.schemePreferences.schemeSelection(civilServant, form))
       }
   }
 
   def submit = CSRSecureAppAction(SchemesRole) { implicit request =>
     implicit user =>
       form.bindFromRequest.fold(
         invalidForm => {
-          Future.successful(Ok(views.html.application.schemePreferences.schemeSelection(invalidForm)))
+          val civilServant = isCivilServant(user.application.fastPassDetails)
+          Future.successful(Ok(views.html.application.schemePreferences.schemeSelection(civilServant, invalidForm)))
         },
         selectedSchemes => {
-          (for {
+          user.application
+          for {
             _ <- schemeClient.updateSchemePreferences(selectedSchemes)(user.application.applicationId)
             redirect <- refreshCachedUser().map(_ => Redirect(routes.PartnerGraduateProgrammesController.present()))
           } yield {
             redirect
-          })
+          }
         }
     )
   }
+
+  private def isCivilServant(fastPassDetails: Option[FastPassDetails]) = {
+    val fp = fastPassDetails flatMap (_.fastPassType)
+    fp.contains("CivilServant") || fp.contains("CivilServantViaFastTrack")
```

Comment:
you can use these literals "CivilServant" "CivilServantViaFastTrack" directly from FastPassForm